source/_integrations/unifi.markdown

```diff
@@ -62,7 +62,15 @@ It is recommended that you run the UniFi Network application in a dedicated virt
 
 ## Button
 
-Provides the ability to restart a UniFi device. This feature requires admin privileges.
+Button entities requires admin privileges to use.
```

Comment:
```suggestion
The button entities listed below require administrator privileges on the UniFi device.
```

Reply:
I don't think this is an improvement as it is for all button entities from the integration, no need to confuse by referencing what is listed.

Reply:
Haha, OK, I see your point. To me, it sounded so generic that for a moment, I thought you're saying _All buttons in Home Assistant_ require administrator privileges. This probably sounds absurd to you, but I actually went and checked if something had changed there.

Reply:
```suggestion
UniFi Network button entities requires admin privileges to use.
```
So maybe like this?

Reply:
```suggestion
The Button entities will only be available and usable if the integration has an UniFi Network account with administrator privileges.
```

Reply:
Love it. So you want me to commit this and merge? Or are you still thinking about it? Asking because you pushed the other commits, but not this one.